lib/vsc/utils/affinity.py

```diff
@@ -238,7 +238,7 @@ def sched_getcpu():
 #    tobin not used anymore
 def tobin(s):
     """Convert integer to binary format"""
-    ## bin() missing in 2.4
+    # bin() missing in 2.4
```

Comment:
the remakr above it says 'tobin not used anymore'
i'll grep code and then remove it